form_validation/built-in_valid/signup.html

```diff
@@ -26,21 +27,130 @@
             </div>
         </header>
 
+        <!-- hidden div with tracking stats -->
+        <div id="hidden-div" style="display: none;" class="info_box">
+            <h2>Behavioral data</h2>
+            <p id="track-clicks">Number of mouse clicks: 0</p>
```

Comment:
i think there is a way to put a placeholder in (so it updates automatically, something with template engine) but I have not learnt it.... :(

Reply:
I will try learn it on Tuesday, if I figure it out, I will write an update